doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.isolatedStarAssociation.IsolatedStarAssociationTask.rst

```diff
@@ -0,0 +1,44 @@
+.. lsst-task-topic:: lsst.pipe.tasks.isolatedStarAssociation.IsolatedStarAssociationTask
+
+###########################
+IsolatedStarAssociationTask
+###########################
+
+``IsolatedStarAssociationTask`` matches unresolved stars from the ``sourceTable_visit`` datasets into a pure (but not complete) catalog of isolated stars.
+The output datasets are a catalog of isolated stars (dataset ``isolated_star_cat``) and the associated observations with configured columns from the input catalogs (dataset ``isolated_star_observations``).
+
+.. _lsst.pipe.tasks.isolatedStarAssociation.IsolatedStarAssociationTask-summary:
+
+Processing summary
+==================
+
+``IsolatedStarAssociationTask`` reads in all the visit-level source table parquet catalogs in a given tract and matches (associates) the observations together.
+In particular:
+
+- All ``sourceTable_visit`` catalogs overlapping a given tract are used as inputs.
+  The configuration variable ``extra_columns`` can be used to specify which columns from the inputs are persisted (in addition to the default flux column, input row number, and source id).
+- Unflagged, unresolved stars above a configured signal-to-noise are selected from each input catalog.
+- The input stars are associated within a configured ``match_radius``.
+  This association is done band-by-band (with the order configured by ``band_order``), and then each band is matched such that all observations are associated with a unique list of stars which cover all the bands in the tract.
+- The star catalog is matched against itself and group of stars that matches neighbor(s) within ``isolation_radius`` is removed from the final isolated star catalog (``isolated_star_cat``).
```

Comment:
"and group of stars that matches" - I think something's not right with the grammar here, but I can't figure out what.